packages/timeline-state-resolver/src/integrations/vizMSE/index.ts

```diff
@@ -846,1496 +905,3 @@ export class VizMSEDevice extends DeviceWithState<VizMSEState, DeviceOptionsVizM
 		this._vizmseManager.ignoreAllWaits = true
 	}
 }
-class VizMSEManager extends EventEmitter {
```

Comment:
Big fan of refactoring this out into other files.
So much a fan that I don't mind fixing the merge-conflicts this will make with my current task :D

Reply:
Only downside is that Git shows all the files as "new file added", so we can't see if you just moved the code or if you also made changes to it

Reply:
Which would be the optimal time to secrectly add your code for world domination ;)